### ISSUE TITLE
No empty values in the log

### ISSUE THREAD

**Issue by Karmalakas** [itsgoingd/clockwork#507]
Using Grav CMS added some dumps to Twig templates and here are my findings (also [original issue created there](https://github.com/getgrav/grav/issues/3393) - was told it's a Clockwork issue):

---

Wasted quite a bit of time until realized debugger doesn't work as expected. Some empty values are completely ignored and not displayed in log. Eg.:
```twig
    {{ dump('TRUE') }}
    {{ dump(true) }}
    {{ dump('FALSE') }}
    {{ dump(false) }}
    {{ dump(0) }}
    {{ dump('') }}
    {{ dump(null) }}
    {{ dump([]) }}
    {{ dump({}) }}
```

Debugger output:
![image](https://user-images.githubusercontent.com/3602819/124160461-f253c500-daa4-11eb-8c16-611068210553.png)

So basically these three are never displayed:
```twig
    {{ dump(false) }}
    {{ dump(0) }}
    {{ dump('') }}
```

P. S. Also was always wondering why `null` is wrapped into an array 🤔

**Comment by mahagr** [comment]
Maybe it's better to add example with clockwork code. :)

**Comment by Karmalakas** [comment]
I have no idea how to do this :)

**Comment by itsgoingd** [comment]
Hey, this seems to work fine when using the Clockwork logging api directly:

```php
    clock('TRUE');
    clock(true);
    clock('FALSE');
    clock(false);
    clock(0);
    clock('');
    clock(null);
    clock([]);
```

<img width="766" alt="Screenshot 2021-07-01 at 22 21 30" src="https://user-images.githubusercontent.com/821582/124185287-102c2480-dabb-11eb-8583-bd7ad7ecca19.png">

This also works fine via Laravel's logger, except in the case of an array, where Laravel already turns the value into a string.

Unfortunately I have no idea how logging is set up in Grav. I've tried taking a brief look into the code, but I would need to spend way more time getting familiar with their code, which is beyond the scope of support I can offer.

**Comment by Karmalakas** [comment]
Hmm... 
`$this->clockwork` is the class of `vendor/itsgoingd/clockwork/Clockwork/Clockwork.php`

```php
$label = 'debug';

$this->clockwork->log($label, 'START');

$this->clockwork->log($label, 'TRUE');
$this->clockwork->log($label, true);
$this->clockwork->log($label, 'FALSE');
$this->clockwork->log($label, false);
$this->clockwork->log($label, 0);
$this->clockwork->log($label, '');
$this->clockwork->log($label, null);
$this->clockwork->log($label, []);
$this->clockwork->log($label, new \stdClass());

$this->clockwork->log($label, 'FINISH');
```

And here's what I get:
![image](https://user-images.githubusercontent.com/3602819/124817339-575e5d80-df72-11eb-962a-aafca4d08970.png)

So it looks like an issue really is in Clockwork

**Comment by itsgoingd** [comment]
Thanks a lot for digging into this, you were indeed correct about the source of the issue. This has been fixed in Clockwork 5.1.

**Comment by Karmalakas** [comment]
cc: @mahagr 
This should be upgraded on Grav at some point 🙂
